packages/@vuepress/core/lib/webpack/createBaseConfig.js

```diff
@@ -39,7 +39,7 @@ module.exports = function createBaseConfig ({
     .output
       .path(outDir)
       .filename(isProd ? 'assets/js/[name].[chunkhash:8].js' : 'assets/js/[name].js')
-      .publicPath(isProd ? publicPath : '/')
+      .publicPath(publicPath)
```

Comment:
What's the reason of changing it?

Reply:
https://github.com/vuejs/vuepress/issues/1238

Reply:
you can reproduce it by this repo
https://github.com/BuptStEve/vuepress-bug